src/Aspirate.Processors/PlaceholderSubstitutionStrategies/ResourceBindingsSubstitutionStrategy.cs

```diff
@@ -35,35 +47,35 @@ public void Substitute(KeyValuePair<string, string> placeholder, Dictionary<stri
         var newValue = bindingProperty switch
         {
             "host" => resourceName,  // return the name of the resource for 'host'
-            "port" => binding.ContainerPort.ToString(),
+            "port" => binding.TargetPort.ToString(),
```

Comment:
We neeed to support scheme, host, port, targetPort and url. There are some rules around how port gets inferred from target port, but only if there's no port specified.